test/tests/api/v2_0/sel_alert_poller_tests.py

```diff
@@ -188,10 +191,12 @@ def test_full_sel(self):
             self.__amqp_alert = {}
             self.run_ipmitool_command((n['node_ip']), "sel add /tmp/selError.txt")
             #time.sleep(1)
+            self.__event_count = 0
+            self.__event_total = n["available_sel_entries"]
+
             self.__task_worker.start()
 
-            assert_equal(len(self.__amqp_alert["value"]["alerts"]), n["available_sel_entries"])
-            #self.__amqp_alert = {}
+            assert_equal(self.__event_count, n["available_sel_entries"])
 
```

Comment:
If SEL entry is added N entrys by ipmitool cmd, the poller is polling in 1 minute interval. Will this lead to N times AMQP alert?